apps/dashboard/src/main/java/com/akto/action/testing/StartTestAction.java

```diff
@@ -611,23 +614,27 @@ public String fetchMetadataFilters() {
     // needed for new ui as the table was server table.
     public String computeAllTestsCountMap(){
         Map<String,Long> result = new HashMap<>();
-
-        long totalCount = TestingRunDao.instance.getMCollection().countDocuments();
-        List<Bson> filtersForCiCd = new ArrayList<>();
-        filtersForCiCd.add(Filters.in(Constants.ID, getCicdTests()));
-        long cicdCount = TestingRunDao.instance.getMCollection().countDocuments(Filters.and(filtersForCiCd));
-
-        int startTime = Context.now();
-        int endTime = Context.now() + 86400;
-        List<Bson> filtersForSchedule = new ArrayList<>();
-        Collections.addAll(filtersForSchedule,
-                Filters.lte(TestingRun.SCHEDULE_TIMESTAMP, endTime),
-                Filters.gte(TestingRun.SCHEDULE_TIMESTAMP, startTime),
-                Filters.nin(Constants.ID,getCicdTests())
-        );
-        long scheduleCount =  TestingRunDao.instance.getMCollection().countDocuments(Filters.and(filtersForSchedule));
-
-        long oneTimeCount = totalCount - cicdCount - scheduleCount;
+        ArrayList<Bson> filters = new ArrayList<>();
```

Comment:
refractor code from retrieveAllCollectionTests and use that here as well for these queries.